os/include/netdb.h

```diff
@@ -331,6 +339,9 @@ void freeaddrinfo(FAR struct addrinfo *ai);
 
 /**
  * @brief gethostbyaddr() is the function returns a corresponding hostname with given IP addresses
+ *
+ * @details @b #include <netdb.h>\n
+ * SYSTEM CALL API
```

Comment:
please check if this API is SYSTEM CALL or not.

Reply:
revised